src/main/kotlin/wu/seal/jsontokotlin/JsonSchemaDataClassGenerator.kt

```diff
@@ -0,0 +1,73 @@
+package wu.seal.jsontokotlin
+
+import com.squareup.kotlinpoet.*
+import wu.seal.jsontokotlin.bean.jsonschema.JsonObjectDef
+
+class JsonSchemaDataClassGenerator(private val jsonObjectDef: JsonObjectDef) {
+  val classes = mutableListOf<TypeSpec>()
+
+  fun generate(className: String?) {
+    val clazz = className
+        ?: jsonObjectDef.title
+        ?: throw IllegalArgumentException("className cannot be null when jsonObjectDef.title is null")
+
+    val requiredFields = jsonObjectDef.required ?: emptyArray()
+    val properties = if (ConfigManager.isOrderByAlphabetical) jsonObjectDef.properties.toSortedMap() else jsonObjectDef.properties
+    val s = TypeSpec.classBuilder(clazz).apply {
+      if (!ConfigManager.isCommentOff && (jsonObjectDef.description?.isNotBlank() == true)) {
```

Comment:
Here may no need to judge the config , the `isCommentOff`　function was implement by interceptor, so threre is no need to deal with option function here.

Reply:
Just add the comment, no need to check config option

Reply:
The comment is presented as Kotlin Doc(multiple lines) in this case, and the interceptor did not deal with the Kotlin Doc by now

Reply:
Got it! now need to check if  code parser could work ok with this new type comment
https://github.com/wuseal/JsonToKotlinClass/blob/master/src/main/kotlin/wu/seal/jsontokotlin/utils/classblockparse/ClassCodeParser.kt

Reply:
@wuseal  I'll try to put this enhancement into the next version since time limits